lib/dirac_coarse.cpp

```diff
@@ -141,11 +146,19 @@ namespace quda {
     if (gpu_setup) dirac->createCoarseOp(*Y_d,*X_d,*transfer,kappa,mass,Mu(),MuFactor());
     else dirac->createCoarseOp(*Y_h,*X_h,*transfer,kappa,mass,Mu(),MuFactor());
 
+    printfQuda("ESW: About to run createYhat\n");
```

Comment:
Remove this print statements

Reply:
I elected to add verbosity guards and make the debug statements more descriptive (and remove the `ESW: `) instead. I found them useful during development and debugging.

Reply:
I also used this as a reminder to get rid of/edit out all `ESW` comments